cmd/geth/log_dispatch.go

```diff
@@ -81,6 +82,8 @@ func mustGetDisplaySystemFromName(s string) displayEventHandlers {
 		return greenDisplaySystem
 	case "dash":
 		return dashDisplaySystem
+	case "gitlike":
```

Comment:
Yea... there's [a new opt-in format for logging](https://github.com/ethereumproject/go-ethereum/pull/619/files#diff-dd0529ce3ceed312f65df1cd264066ca).